### ISSUE TITLE
Request a way to pass other options to lame

### ISSUE THREAD

**Issue by rsteckler** [TooTallNate/node-lame#51]
Firstly - thanks for the project.  I'm running it on a Raspberry pi without issue!

I'm trying to pass --scale (or lame_set_scale) to the encoder.  I've been looking at the source for about an hour, but it's hard to follow and I'm not able to get it running in the debugger to see what's really going on.  It looks like it's *almost* able to pass arbitrary options (via the opts structure, to the get/set loop at the bottom of encoder.js, to the gfp structure, the the c++ code).  I'm missing something in the link between those things though.  
Is this possible?

Thanks!
Ryan

**Comment by ssegeo** [comment]
@rsteckler , did find a workaround this, I am also in the same situation, I wish to set additional options i.e title, artist, albumArt for the song. Mind pointing in the right direction if possible

**Comment by rsteckler** [comment]
This was a long time ago.  I don't think I ever figured it out.

On Sat, Feb 13, 2016 at 5:44 AM ssegeo <notifications@github.com> wrote:

> @rsteckler <https://github.com/rsteckler> , did find a workaround this, I
> am also in the same situation, I wish to set additional options i.e title,
> artist, albumArt for the song. Mind pointing in the right direction if
> possible
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TooTallNate/node-lame/issues/51#issuecomment-183667464>
> .
>
-- 

Thanks,
Ryan Steckler
